pkg/api/v20170930/types.go

```diff
@@ -0,0 +1,16 @@
+package v20170930
+
+// VersionInfo contains rersion and release numbers
+type VersionInfo struct {
+	Release string `json:"release,omitempty"`
```

Comment:
Why not be consistent and use?:
OrchestratorRelease string            `json:"orchestratorRelease"`
OrchestratorVersion string            `json:"orchestratorVersion" validate:"len=0"`